src/extension/background-script/actions/nostr/decryptOrPrompt.ts

```diff
@@ -18,10 +18,9 @@ const decryptOrPrompt = async (message: MessageDecryptGet) => {
     );
 
     if (hasPermission) {
-      const response = (await state.getState().getNostr()).decrypt(
-        message.args.peer,
-        message.args.ciphertext
-      );
+      const response = await (
```

Comment:
I think we should do this in two lines

const nostr = await state.getState().getNostr();
and then 
const response = await nostr.decrypt(message.args.peer, message.args.ciphertext);

this makes this easier to read and more obvious. 

and I think it prevents that some await is missed.